scripts/checks.sh

```diff
@@ -19,11 +19,9 @@ sls remove
 popd
 
 
-pushd integration-test
```

Comment:
No need for `cd` to another directory because the venv is saved in the miniconda foder:
`environment location: /home/circleci/miniconda/envs/myvenv`